modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2TreeIndexBase.java

```diff
@@ -38,8 +41,15 @@
  * H2 tree index base.
  */
 public abstract class H2TreeIndexBase extends GridH2IndexBase {
+
     /** Default value for {@code IGNITE_MAX_INDEX_PAYLOAD_SIZE} */
-    static final int IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT = 10;
+    static final int IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT = 64;
```

Comment:
need to be carefully and understand that effective size will be less by 3. May be good to add such comment here.

Reply:
Yes, just wrote the same in my comment. Does it worth to add extra bytes to provided IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT  in computeInlineSize() to be sure that it will be working with any value?

Reply:
why do you choose 64 for default?

Reply:
Just found "Change of index inline defaults" thread, so question is not relevant any more

Reply:
Guys, what do you think about increasing of the actual index size by adding of 3 bytes to the IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT value? This property can be changed by user, so it will be more safe to add them by ourselves.

Reply:
Also what do you think about increasing of IGNITE_VARIABLE_TYPE_DEFAULT_INDEX_SIZE to 13 (3 bytes are taken for our info)?

Reply:
>  what do you think about increasing of the actual index size by adding of 3 bytes to the IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT value?

I think it should be left as is (64) because we have no information about types of the fields the index will consists of.

> Also what do you think about increasing of IGNITE_VARIABLE_TYPE_DEFAULT_INDEX_SIZE to 13 

The same. Let it be 10 as it was discussed previously with the arch-team.

Reply:
Ok, but note that IGNITE_MAX_INDEX_PAYLOAD_SIZE_DEFAULT can be set by user. I think that we should add at least some assertion.